now.json

```diff
@@ -1,10 +1,10 @@
 {
-  "name": "vuetifyjs.com",
```

Comment:
This isn't deprecated?

Reply:
When running `now` it says it is
![now](https://user-images.githubusercontent.com/27827979/76165516-02b49380-612e-11ea-9e37-2f4e59c862a7.png)

Reply:
Yeah I got that after updating now-cli.